### ISSUE TITLE
[Bug] Can't move bag slots around, can't put new bags into bank slots

### ISSUE THREAD

**Issue by penguinsane** [failcoder/SVUI-Kit#68]
Same issue as was happening before off and on, happening again.

Can sometimes move bag slots around, but it's hard to actually put new bags into slots, especially in the bank. Have to disable bag module to do it.

**Comment by failcoder** [comment]
Nothing has changed in bags though. Lets try this, tell me EXACTLY what you did when it happened (ie... what specific bag it was, was the bag slot empty prior to the attempt, what was the actual outcome?).

**Comment by penguinsane** [comment]
Purchased first two bank bag slot, attempted to move already-soulbound Netherweave Bag into empty bank slot. Drag bag over, attempting to click... nothing happens. No error, just doesn't happen.

For the most part, it seems like the regular inventory is working just fine. I seem to be able to move bags around, take them off, put them back.

The bank is where this problem is right now.

Made a gif for you of what's happening: http://puu.sh/g1szP/bec43e7e57.gif

**Comment by failcoder** [comment]
BOOYA!! Fixed!!!!

**Comment by penguinsane** [comment]
Hrm. Fresh install, deleted all SVUI-related .WTF files, still can't drag my Netherweave Bag into my available bank slot. :(

**Comment by failcoder** [comment]
You are KILLING me smalls!
Updated, try it now.

**Comment by penguinsane** [comment]
Well I'm sorry! I'm just trying to do what you keep me around for, lol.

Updated, no change. (Don't hurt me.)

**Comment by Nyrin** [comment]
I am now having this problem. i can take stuff out of my bank bags, remove the bags from the slot, but i can not equip the bag in the bank slot.

**Comment by Nyrin** [comment]
had to disable all of svui to add the bags to bank slots.

**Comment by hotluck** [comment]
They were still showing as blank and gave the following error -

Date: 2015-03-05 11:28:41
ID: 2
Error occured in: Global
Count: 1
Message: ..\AddOns\SVUI_!Options\UnitFrames.lua line 2433:
   attempt to index field '?' (a string value)
Debug:
   (tail call): ?
   SVUI_!Options\UnitFrames.lua:2433:
      SVUI_!Options\UnitFrames.lua:2432
   (tail call): ?
   [C]: ?
   [string "safecall Dispatcher[2]"]:9:
      [string "safecall Dispatcher[2]"]:5
   (tail call): ?
   ...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:799:
      ...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:614
   (tail call): ?
   [C]: ?
   [string "safecall Dispatcher[3]"]:9:
      [string "safecall Dispatcher[3]"]:5
   (tail call): ?
   ...\AddOns\SVUI_!Options\libs\AceGUI-3.0\AceGUI-3.0.lua:314: Fire()
   ...s\AceGUI-3.0-SharedMediaWidgets\BackgroundWidget.lua:23:
      ...s\AceGUI-3.0-SharedMediaWidgets\BackgroundWidget.lua:21
Locals:
None

I have looked in the wtf file and found that under SVUI_Media, there is no profile for Main Level (100) which is the profile I am using so I think this is the problem. This is probably from the various other profile problems I have been testing and you have fixed. I think the data has just got corrupted. I could just copy that bit from one of the other profiles but that could cause more problems, so I am just going to delete the wtf files and start again from scratch (you've done so many fixes and changes since I first did it that it's the best option I think).

I'll let you know when I've done it.
